timon/db/serializers.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.db.serializers
+"""
+Summary: Serializers and some db funcs that cannot be imported before
+the db configuration
+"""
+# #############################################################################
+import logging
+from datetime import datetime
+from threading import Thread
+
+from timon.conf.flags import FLAG_UNKNOWN_STR
+from timon.db.models import ProbeRslt
+
+logger = logging.getLogger(__name__)
+
+
+def init_db(db):
+    db.connect()
+    db.create_tables([ProbeRslt])
+
+
+class PeeweeDbThreadingStore(Thread):
+    """
+    A thread that permits to write probe results in a peewee db at
+    regular intervals.
+    For every probe, it have only 10 records and every time it receive
+    a new record for a probe, it overwrite the older one to keep this
+    number of 10 records.
+    """
+    def __init__(self, backend, probenames):
+        self.backend = backend
+        self.rsltqueue = backend.rsltqueue
+        self.stopevent = backend.stopevent
+        self.interval = 60
+        self.chunk_size = 100
+        self.stored_records = 10
+        self.probenames = probenames
+        super().__init__()
+
+    def init_db(self):
+        """
+        If don't exist, creates 10 fake probe results for every probenames
+        """
+        logger.info("Init db")
+        with self.backend.db.transaction() as transaction:
+            chunk = 0
```

Comment:
rename to chunk_cnt and add commenty
e.g.
# commit transaction for every self.chunk_size created elements